test/integration/framework/util.go

```diff
@@ -62,35 +60,7 @@ func DeleteNamespaceOrDie(c clientset.Interface, ns *v1.Namespace, t *testing.T)
 	}
 }
 
-// GetReadySchedulableNodes addresses the common use case of getting nodes you can do work on.
-// 1) Needs to be schedulable.
-// 2) Needs to be ready.
-// If EITHER 1 or 2 is not true, most tests will want to ignore the node entirely.
-// If there are no nodes that are both ready and schedulable, this will return an error.
-func GetReadySchedulableNodes(c clientset.Interface) (nodes *v1.NodeList, err error) {
-	nodes, err = checkWaitListSchedulableNodes(c)
-	if err != nil {
-		return nil, fmt.Errorf("listing schedulable nodes error: %s", err)
-	}
-	Filter(nodes, func(node v1.Node) bool {
-		return IsNodeSchedulable(&node) && isNodeUntainted(&node)
-	})
-	if len(nodes.Items) == 0 {
-		return nil, fmt.Errorf("there are currently no ready, schedulable nodes in the cluster")
-	}
-	return nodes, nil
-}
-
-// checkWaitListSchedulableNodes is a wrapper around listing nodes supporting retries.
-func checkWaitListSchedulableNodes(c clientset.Interface) (*v1.NodeList, error) {
-	nodes, err := waitListSchedulableNodes(c)
-	if err != nil {
-		return nil, fmt.Errorf("error: %s. Non-retryable failure or timed out while listing nodes for integration test cluster", err)
-	}
-	return nodes, nil
-}
-
-// waitListSchedulableNodes is a wrapper around listing nodes supporting retries.
+// waitListSchedulableNodes is a wrapper around listing schedulable nodes supporting retries.
 func waitListSchedulableNodes(c clientset.Interface) (*v1.NodeList, error) {
```

Comment:
let's just get rid of this function

Reply:
You mean use `waitListAllNodes` below instead?

Reply:
yes

Reply:
updated and squashed.